src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -444,6 +490,25 @@ private static boolean hasLineBreakBefore(DetailAST rightCurly) {
     }
 
     /**
+     * Checks if right curly is alone on line in token the lambda.
+     * @param details for validation.
+     * @return true, if right curly is alone on line.
+     */
+    private static boolean isAloneOther(Details details) {
```

Comment:
Method name is very controversial. I try to read javadoc comment but it has nothing in common with the name.